src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -173,27 +173,51 @@
    constants/optimism-chain-id :optimism
    constants/arbitrum-chain-id :arbitrum})
 
+(defn calculate-fiat-change
+  "Returns unsigned fiat change"
+  [fiat-value change-pct]
+  (-> (if (number? change-pct) change-pct 0)
+      money/bignumber
+      (money/div 100)
+      (money/mul fiat-value)
+      money/absolute-value))
+
+(defn prettify-percentage-change
+  "Returns unsigned precentage"
+  [percentage]
+  (-> (if (number? percentage) percentage 0)
+      money/bignumber
+      money/absolute-value
+      (money/to-fixed 2)))
+
 (defn calculate-token-value
   "This function returns token values in the props of token-value (quo) component"
   [{:keys [token color currency currency-symbol]}]
-  (let [token-units                 (total-token-units-in-all-chains token)
-        fiat-value                  (total-token-fiat-value currency token)
-        market-values               (or (get-in token [:market-values-per-currency currency])
-                                        (get-in token
-                                                [:market-values-per-currency
-                                                 constants/profile-default-currency]))
-        {:keys [change-pct-24hour]} market-values
-        crypto-value                (get-standard-crypto-format token token-units)
-        fiat-value                  (if (string/includes? crypto-value "<")
-                                      "<$0.01"
-                                      (prettify-balance currency-symbol fiat-value))]
+  (let [token-units               (total-token-units-in-all-chains token)
+        fiat-value                (total-token-fiat-value currency token)
+        market-values             (or (get-in token [:market-values-per-currency currency])
+                                      (get-in token
+                                              [:market-values-per-currency
+                                               constants/profile-default-currency]))
+        {:keys [change-pct-hour]} market-values
+        has-token-units?          (money/greater-than token-units 0)
+        fiat-change               (calculate-fiat-change fiat-value change-pct-hour)
+        formatted-fiat-change     (prettify-balance currency-symbol fiat-change)
+        percentage-change         (prettify-percentage-change change-pct-hour)
+        crypto-value              (get-standard-crypto-format token token-units)
+        formatted-fiat-value      (if (string/includes? crypto-value "<")
+                                    "<$0.01"
+                                    (prettify-balance currency-symbol fiat-value))]
```

Comment:
Why do we depend on a string containing a `<`?

Seems we should compare the values directly instead of using the result of `get-standard-crypto-format`.

Reply:
@ulisesmac If the crypto-value includes `<` it means its value is less than one USD cent. In that case we display "<$0.01" for fiat.

Reply:
@OmarBasem
Yeah, but I wanted to say that we should instead compare that the value us less than 0.01, instead of checking if `<` is in the string.

The reason is the logic is more clear and we wouldn't rely on a formatted string that is expected to be consumed by the user. If designers decide to change `<` for `less than` it should only affect UIs

Reply:
I was checking with Desktop team yesterday about that. We need to get the formatting from status-go. We can create an issue for refactoring in Feb.

Reply:
OK! thanks! :)